docs/ops/arithmetic/Tanh_1.md

```diff
@@ -6,15 +6,21 @@
 
 **Short description**: Tanh element-wise activation function.
```

Comment:
Let's rename it to "unary arithmetic function". Refer to https://github.com/openvinotoolkit/openvino/pull/5369/